kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/PodOperationsImpl.java

```diff
@@ -55,13 +55,13 @@ public String getLog(String containerId) {
   @Override
   public String getLog(String containerId, Boolean isPretty) {
     StringBuilder sb = new StringBuilder();
-    sb.append("log?pretty=").append(isPretty);
+    sb.append("/log?pretty=").append(isPretty);
     if (containerId != null && !containerId.isEmpty()) {
       sb.append("&container=").append(containerId);
     }
 
     try {
-      URL url = new URL(getResourceUrl(), sb.toString());
```

Comment:
Why didn't this work? I assume because `getResourceUrl` didn't end in a slash? You probably should check that here, sometimes `//` will cause issues.